src/app/components/WorkDetail/DefinitionList.jsx

```diff
@@ -32,6 +32,8 @@ export const DefinitionList = (props) => {
   const newSearchRequest = (event, query, field) => {
     event.preventDefault();
 
+    props.dispatch(userQuery(query));
+    props.dispatch(selectedField(field));
```

Comment:
I don't know what the best practice around this is, whether to just trigger one action per event or multiple actions per event.

Reply:
It's probably better to add a reset action to change state to empty values where appropriate. Am not sure how that will work with Redux, but will add a new action/reducer to handle a reset instead of calling three actions.

Reply:
Yes, a reset action/reducer has worked for me in the past. I think we've called the reset action _after_ the initial action has finished.